desktop/apps/consignments2/components/thank_you/index.js

```diff
@@ -8,9 +8,9 @@ import block from 'bem-cn'
 import { connect } from 'react-redux'
 import { get } from 'lodash'
 
-function ThankYou ({ submission, uploadedImages }) {
+function ThankYou ({ submission, processingImages, uploadedImages }) {
   const b = block('consignments2-submission-thank-you')
-  const submissionImage = get(uploadedImages, '0.src') ? get(uploadedImages, '0.src') : '/images/missing_image.png'
+  const submissionImage = get(uploadedImages, '0.src') && processingImages.length === 0 ? get(uploadedImages, '0.src') : '/images/missing_image.png'
```

Comment:
I would either a) `import get from 'lodash.get'` since lodash is not located in `package.json` (this is important in terms of proper dependency management https://github.com/artsy/force/blob/master/package.json#L81) or b) import it off of underscore https://github.com/artsy/force/blob/master/desktop/lib/global_client_setup.coffee#L10. The reason why this works is `lodash` is a `devDependency` of some other lib and the lookup path for imports includes `node_modules`, but strictly speaking this is not correct usage and could break the chain at some further point in time. 

(This reminds me that I need to move this `_.get =` setter up a level into the main lib, independent of client or server.)